docs/api/profiles.md

```diff
@@ -0,0 +1,86 @@
+# Profiles
+
+Profiles contain information about a User. A User does not need a Profile, but a Profile is always created
+together with a User.
```

Comment:
When creating a profile for a user that already has a profile, an error will be returned (status 409 with a json message stating the profile exists).

Url for the current user: `/user/profile`
Url for another user: `/users/:userid/profile`